lib/query-interface.js

```diff
@@ -1122,8 +1122,8 @@ class QueryInterface {
     }
   }
 
-  createFunction(functionName, params, returnType, language, body, options) {
```

Comment:
Since you are using functions, can you add proper comments for this method options and usage, something like https://github.com/sequelize/sequelize/blob/master/lib/query-interface.js#L157

Reply:
done

Reply:
Great :+1: